slices/libapache2-mod-php8.3.yaml

```diff
@@ -0,0 +1,63 @@
+package: libapache2-mod-php8.3
+
+essential:
+  - php8.3-common_copyright
+
+slices:
+  apache2-config:
+    essential:
+      - apache2_config
+      - libapache2-mod-php8.3_libs
+    contents:
+      /etc/apache2/mods-available/php8.3.conf:
+      /etc/apache2/mods-available/php8.3.load:
+      /etc/apache2/mods-enabled/php8.3.conf:
+        symlink: /etc/apache2/mods-available/php8.3.conf
+      /etc/apache2/mods-enabled/php8.3.load:
+        symlink: /etc/apache2/mods-available/php8.3.load
+
+  config:
+    essential:
+      - apache2-bin_bins
```

Comment:
I think the `libs` slice should have `apache2-bin_bins` in it's essential list, not the `config` slice.
Additionally, I believe the `config` slice should have the `libs` slices in it's essential list.